test/index.test.js

```diff
@@ -238,30 +252,48 @@ describe('AdRoll', function() {
         it('should pass custom data like product id and sku', function() {
           analytics.track('event', { revenue: 2.99, id: '12345', sku: '43434-21', other: '1234' });
           analytics.called(window.__adroll.record_user, {
-            adroll_segments: 'segment',
+            adroll_segments: 'mappedVal',
             adroll_conversion_value_in_dollars: 2.99,
             product_id: '12345',
             sku: '43434-21',
-            adroll_custom_data: { other: '1234' },
+            other: '1234',
             order_id: '12345'
           });
         });
+      });
 
-        it('should support array events', function() {
-          adroll.options.events = [{ key: 'event', value: 'pixel' }];
-          analytics.track('event', { total: 2.99, orderId: 2 });
-          analytics.called(window.__adroll.record_user, {
-            adroll_segments: 'pixel',
-            adroll_conversion_value_in_dollars: 2.99,
-            order_id: 2
+      describe('Event Not Mapped', function() {
+        describe('V1', function() {
+          beforeEach(function() {
+            adroll.options._version = 1;
+          });
+
+          it('should snake_case and send unmapped events', function() {
+            analytics.track('Event A');
+            analytics.called(window.__adroll.record_user, {
+              adroll_segments: 'event_a'
+            });
+          });
+
+          it('should map revenue and order id', function() {
```

Comment:
Seems like this test is duplicated above in `should pass in revenue and order id`. Break these tests apart into smaller tests; "should map revenue and order id" can likely be two individual tests. At that point your mapped test suite will just be a matter of testing whether or not the event will be renamed, which I'd expect will increase the amount of shared tests we have and make this a lot easier to maintain, which will quickly become a problem given the logic branch we've introduced by adding versioning.